libraries/core/helpers/version.js

```diff
@@ -19,8 +19,8 @@ export const defaultClientInformation = {
   libVersion: '17.0',
   appVersion: '5.18.0',
   codebaseVersion: '5.18.0',
-  type: (!md.tablet() ? 'phone' : 'tablet'),
```

Comment:
Is it safe for current implementations. Nothing is expecting this already? Should be maybe keep both?